src/components/date/date.js

```diff
@@ -647,6 +647,10 @@ const Date = Input(InputIcon(InputLabel(InputValidation(class Date extends React
    * @return {String} formatted visible value
    */
   formatVisibleValue(value) {
+    if (value === '') {
```

Comment:
why do you not also add a prop to the component to allow a consumer / developer to choose which behaviour they want ?

your change may break existing code using this component that which expects the date to automatically be `today` when there is no value

Reply:
@LPTSage You're right. I updated the code.